cfgov/jinja2/v1/_includes/molecules/contact-email.html

```diff
@@ -23,7 +24,7 @@
     <p>
         {% for email in value.emails %}
```

Comment:
Not really related to this PR, but do we really need the ability to have multiple emails within a single ContactEmail when one can just add multiple ContactEmails to a Contact (which is what we do in at least the first example I looked at, Accessibility Feedback)?

Reply:
I was wondering about that too. I'd bet the purpose is to allow for the listing of multiple addresses without repeating the "✉️ **EMAIL**" header:

![image](https://user-images.githubusercontent.com/654645/71003791-4065cb00-20af-11ea-8166-c165ef20dc79.png)

Reply:
Yeah, I guess that is useful. But we should use it, then :)